### PR TITLE
Wallet: Add Pagination to Transactions View

### PR DESCRIPTION
**Pull request by figitaki**
Adds pagination to the Transactions View, also cleans up some various styles across the app.

### CODE REVIEW

File: frontend/wallet/src/render/components/AddWalletModal.re [hunk 1]
```diff
@@ -20,18 +18,17 @@ module Styles = {
 let make = (~walletName, ~setModalState, ~onSubmit) => {
   <Modal title="Add Wallet" onRequestClose={() => setModalState(_ => None)}>
     <div className=Styles.container>
+      <Alert
+        kind=`Info
+        message="You can change the name or delete the wallet later."
+      />
```
Comment (figitaki) [inline frontend/wallet/src/render/components/AddWalletModal.re]: Moved this alert to the top of the modal to match other modals.

File: frontend/wallet/src/render/components/Link.re [hunk 2]
```diff
@@ -1,3 +1,8 @@
+type kind =
+  | Blue
+  | Grey
+  | Red;
+
```
Comment (figitaki) [inline frontend/wallet/src/render/components/Link.re]: Refactored the link to have three different styles, with default to Blue. Herold is taking another pass at these styles so they'll likely change.

File: frontend/wallet/src/render/components/Toggle.re [hunk 3]
```diff
@@ -55,7 +55,7 @@ let make = (~value, ~onChange) =>
       type_="checkbox"
       id="checkbox"
       checked=value
-      onClick={_e => onChange(toggle => !toggle)}
+      onChange={_e => onChange(toggle => !toggle)}
```
Comment (figitaki) [inline frontend/wallet/src/render/components/Toggle.re]: Fixes a pesky console error about not using onChange on a checkbox

File: frontend/wallet/src/render/components/TransactionsView.re [hunk 4]
```diff
@@ -38,8 +52,8 @@ module Styles = {
 
 module Transactions = [%graphql
   {|
-    query transactions($publicKey: String!) {
-      blocks(filter: { relatedTo: $publicKey }) {
+    query transactions($after: String, $publicKey: String!) {
+      blocks(first: 5, after: $after, filter: { relatedTo: $publicKey }) {
```
Comment (figitaki) [inline frontend/wallet/src/render/components/TransactionsView.re]: Five blocks at a time seemed reasonable.

File: frontend/wallet/src/render/components/TransactionsView.re [hunk 5]
```diff
@@ -92,14 +110,35 @@ let extractTransactions: Js.t('a) => array(TransactionCell.Transaction.t) =
...
+    Option.map(~f=PublicKey.toString, activeWallet)
+    |> Option.withDefault(~default="");
+  let transactionQuery = Transactions.make(~publicKey="123", ~after="", ());
+  let (isFetchingMore, setFetchingMore) = React.useState(() => false);
+
+  let keyForTransaction = (data, index) =>
+    Option.withDefault(~default="", data##blocks##pageInfo##lastCursor)
+    ++ string_of_int(index);
+
+  let updateQuery: ReasonApolloQuery.updateQueryT = [%bs.raw
```
Comment (figitaki) [inline frontend/wallet/src/render/components/TransactionsView.re]: This is definitely the most gnarly bit, reason-apollo doesn't provide valid bindings for the update function so we have to just bs raw it. :( ref. https://github.com/apollographql/reason-apollo/blob/c027522cf9e3aa30bfcc79f36dc8271b6cc327d9/examples/swapi/src/Persons.re#L44

File: frontend/wallet/src/render/components/Link.re (hunk 2 above)
Comment (Schmavery) [reply frontend/wallet/src/render/components/Link.re]: Any thoughts on using polymorphic variants for these kinds of color things

File: frontend/wallet/src/render/components/TransactionsView.re [hunk 6]
```diff
@@ -92,14 +110,35 @@ let extractTransactions: Js.t('a) => array(TransactionCell.Transaction.t) =
 
 [@react.component]
 let make = () => {
-  let transactionQuery = Transactions.make(~publicKey="123", ());
+  let activeWallet = Hooks.useActiveWallet();
+  let activeWalletKey =
+    Option.map(~f=PublicKey.toString, activeWallet)
+    |> Option.withDefault(~default="");
+  let transactionQuery = Transactions.make(~publicKey="123", ~after="", ());
```
Comment (Schmavery) [inline frontend/wallet/src/render/components/TransactionsView.re]: Should we update this to be the active wallet?

Comment (figitaki) [reply frontend/wallet/src/render/components/TransactionsView.re]: Yeah i'll do this in the pending txn pr

**Pull request merged**